flink/v2.0/flink/src/test/java/org/apache/iceberg/flink/source/TestStreamingReaderOperator.java

```diff
@@ -275,7 +274,6 @@ private OneInputStreamOperatorTestHarness<FlinkInputSplit, RowData> createReader
         StreamingReaderOperator.factory(inputFormat);
     OneInputStreamOperatorTestHarness<FlinkInputSplit, RowData> harness =
         new OneInputStreamOperatorTestHarness<>(factory, 1, 1, 0);
-    harness.getStreamConfig().setTimeCharacteristic(TimeCharacteristic.ProcessingTime);
```

Comment:
don't need to set processing time?

Reply:
That's no longer required. The test harness changed. The change is test only.